index.html

```diff
@@ -1,85 +1,148 @@
-            <li class="css">css</li>
-            <li class="html">html</li>
-         <li class="bootstrap">bootstrap</li>
-             <li class="ruby"> ruby </li>
-           </ul>
-        </div>
-        <button type="submit"> See Project</button>
+          <div class="image">
+            <img class="blank" src="./image/blank-img.png" alt="image-placeholder">
+          </div>
+          <h2>Multi-Post Stories</h2>
+          <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+            industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="list">
+            <ul>
+              <li class="css">css</li>
+              <li class="html">html</li>
+              <li class="bootstrap">bootstrap</li>
+              <li class="ruby"> ruby </li>
+            </ul>
+          </div>
+          <button type="submit"> See Project</button>
         </div>
       </li>
       <li class="multi-story-4">
-        
-<div class="multi-story-4">
-  <img src="./image/blank-img.png" alt="image-placeholder">
-<h2>Multi-Post Stories</h2>
-<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-<div class="list">
-  <ul>
-    <li class="css">css</li>
-    <li class="html">html</li>
- <li class="bootstrap">bootstrap</li>
-     <li class="ruby"> ruby </li>
-   </ul>
-</div>
-<button type="submit"> See Project</button>
-</div>
+
+        <div class="multi-story-4">
+          <div class="image">
+            <img class="blank" src="./image/blank-img.png" alt="image-placeholder">
+          </div>
+          <h2>Multi-Post Stories</h2>
+          <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+            industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="list">
+            <ul>
+              <li class="css">css</li>
+              <li class="html">html</li>
+              <li class="bootstrap">bootstrap</li>
+              <li class="ruby"> ruby </li>
+            </ul>
+          </div>
+          <button type="submit"> See Project</button>
+        </div>
+      </li>
+      <li class="mutli-story-5">
+        <div class="contact-form multi-story-5">
+          <h2>CONTACT US</h2>
+          <p class="text">If you have an application you are interested in developing, a feature that you need built or a project
+            that needs coding. I’d love to help with it </p>
+          <div class="form">
+            <form action="https://formspree.io/f/mwkabepk" method="POST">
+              <div class="name">
+                <input type="text" name="name" class="name" placeholder="input name" maxlength="20" minlength="4"
```

Comment:
- [ ] We're having a validation issue here: based on the project specification, the name input should have a maximum length of 30.

Reply:
- This issue is addressed well! :clap: